bin/kevm

```diff
@@ -391,6 +391,7 @@ kevm_port='8545'
 kevm_host='127.0.0.1'
 [[ ! "${run_command}" == prove     ]] || backend=haskell
 [[ ! "${run_command}" == solc-to-k ]] || backend=haskell
+[[ ! "${run_command}" == view-kcfg ]] || backend=haskell
```

Comment:
Should `view-kcfg` be used with `--backend foundry` when it is used with a foundry proof?

Reply:
No, we by default want this to work with any proof. No good way to detect it's a foundry proof. `foundry-view-kcfg` is the command to use for those ones, which does it's own argument processing.